config/routes.rb

```diff
@@ -298,11 +298,9 @@
     end
   end
 
-  resources :audits, :only => [:index] do
-    collection do
-      get 'auto_complete_search'
```

Comment:
@glekner I'm afraid this causes regression in `app/views/users/_audit_summary_query_builder.html.erb` No method error `auto_complete_search_path`

Reply:
right... it should be 'audits_auto_complete_search'... I'll send a PR.

Reply:
#7089